README.md

```diff
@@ -1,207 +1,207 @@
+
+> Build a Java Web API application using Managed Identity, Key Vault and Cosmos DB that is designed to be deployed to Azure App Service or AKS
 
 ![License](https://img.shields.io/badge/license-MIT-green.svg)
 
 This is a Java Spring Boot Web API reference application designed to "fork and code" with the following features:
 
-- Securely build, deploy and run an App Service (Web App for Containers) application
+- Securely build, deploy and run an Azure App Service (Web App for Containers) application
+- Securely build, deploy and run an Azure Kubernetes Service (AKS) application
 - Use Managed Identity to securely access resources
 - Securely store secrets in Key Vault
-- Securely build and deploy the Docker container from Container Registry
-- Connect to and query CosmosDB
+- Securely build and deploy the Docker container to Azure Container Registry (ACR) or Docker Hub
+- Connect to and query Cosmos DB
 - Automatically send telemetry and logs to Azure Monitor
-- Instructions for setting up Key Vault, ACR, Azure Monitor and Cosmos DB are in the Helium [readme](https://github.com/retaildevcrews/helium)
+
+> Visual Studio Codespaces is the easiest way to evaluate helium as all of the prerequisites are automatically installed
+>
+> Follow the setup steps in the [Helium readme](https://github.com/retaildevcrews/helium) to setup Codespaces
 
 ## Prerequisites
 
-- Azure subscription with permissions to create:
-  - Resource Groups, Service Principals, Keyvault, CosmosDB, App Service, Azure Container Registry, Azure Monitor
-- Bash shell (tested on Mac, Ubuntu, Windows with WSL2)
-  - Will not work in Cloud Shell unless you have a remote dockerd
-  - Will not work in WSL1
+- Bash shell (tested on Visual Studio Codespaces, Mac, Ubuntu, Windows with WSL2)
+  - Will not work in Cloud Shell or WSL1
 - Azure CLI ([download](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest))
 - Docker CLI ([download](https://docs.docker.com/install/))
-- Java 8 above ([download](https://www.java.com/en/download/manual.jsp/))
-- JQ ([download](https://stedolan.github.io/jq/download/))
+- Java 8+ ([download](https://www.java.com/en/download/manual.jsp/))
 - Maven ([download](https://maven.apache.org/download.cgi))
-- Completion of the Key Vault, ACR, Azure Monitor and Cosmos DB setup in the Helium [readme](https://github.com/retaildevcrews/helium)
 - Visual Studio Code (optional) ([download](https://code.visualstudio.com/download))
 
 ## Setup
 
-The application requires Key Vault and Cosmos DB to be setup per the Helium [readme](https://github.com/retaildevcrews/helium)
+- Initial setup instructions are in the [Helium readme](https://github.com/retaildevcrews/helium)
+  - Please complete the setup steps and then continue below
 
-- Fork this repo and clone to your local machine
-  - cd to the base directory of the repo
-  - All instructions assume starting from the root of the repo
+### Validate az CLI works
```

Comment:
I feel like there should be another ## header after Setup and before CI as right now everything is under Setup.  Maybe something like "Run the application" before the codespaces and bash sections ?

Reply:
Done!